stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1703,6 +1703,28 @@ func _TFCOpAddInputFromTensorGroup<T : TensorArrayProtocol>(
   return count
 }
 
+/// Special protocol for calling tensorflow operations that take heterogeneous
+/// arrays as input.
+public protocol AnyTensor {
+  var rawTensorHandle: CTensorHandle { get }
```

Comment:
These internal things should be underscored, right? Any reason an end user should use them or see them in code completion?